### ISSUE TITLE
Add support for Android and iOS

### ISSUE THREAD

**Issue by timrid** [pypa/cibuildwheel#1960]
Are there plans to support Android and iOS wheels as well?

For Python 3.13, iOS ([PEP730](https://peps.python.org/pep-0730/)) and Android ([PEP738](https://peps.python.org/pep-0738/)) will also be supported. It would be practical to be able to easily create wheels for these platforms.

**Comment by henryiii** [comment]
I think that the BeeWare folks plans to help support this in cibuildwheel, I was told later this year at PyCon.

**Comment by freakboy3742** [comment]
Confirming that yes, we (the BeeWare folks) are working on this. I've got an [initial WIP branch with iOS support](https://github.com/freakboy3742/cibuildwheel/tree/ios-support), which is sufficient to get a simple project building, but still needs a lot of work. #2062 is the first piece of that work to be upstreamed.

**Comment by Jzhenli** [comment]
Hi, will Android ([PEP738](https://peps.python.org/pep-0738/))  be supported?

**Comment by freakboy3742** [comment]
@Jzhenli Yes - it's on our (BeeWare's) todo list. We're trying to get iOS support added first, rather than trying to add 2 platforms at once.

**Comment by henryiii** [comment]
iOS merged. Now just Android left. ;)

**Comment by freakboy3742** [comment]
We're working on it!
